modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -2385,14 +2385,20 @@ def from_pandas(cls, df):
         """
         new_index = df.index
         new_columns = df.columns
+
+        if isinstance(new_columns, MultiIndex):
+            index_cols = None
+            ErrorMessage.single_warning(
+                "MultiIndex columns are not currently supported by the HDK backend."
+            )
```

Comment:
I would like to have an in-code comment explaining that this is done because of permanent falling to pandas for this frame or smth similar

Reply:
Done.